Mathlib/Data/Fintype/Card.lean

```diff
@@ -540,6 +540,7 @@ theorem card_pos [h : Nonempty α] : 0 < card α :=
   card_pos_iff.mpr h
 #align fintype.card_pos Fintype.card_pos
 
+@[simp]
```

Comment:
I remember these sort of simp lemmas being slow in Lean3 - it may be worth making a note here to make sure that there's no performance drop here

Reply:
@ericrbg I believe this is now fine but if you (or someone else) wants I can excise this change.

Reply:
I'll leave it, this should be a simp lemma morally